thinkhazard/static/less/common.less

```diff
@@ -35,10 +35,16 @@ ul.horizontal li {
 }
 
 .navbar {
+  display: block;
```

Comment:
It says in the Document under Menu items display: block. So I understood that it is required.